src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -369,14 +377,24 @@ export default defineComponent({
           controlBarItems.splice(index, 1)
         }
 
+        // regardless of what DASH qualities you enable or disable in the qualitLevels plugin
+        // the first segments videojs-http-streamming request are chosen based on the available bandwidth, which is set to 0.5MB/s by default
+        // overriding that to be the same as the quality we requested, makes videojs-http-streamming pick the correct quality
+        const playerBandwidthOption = {}
+
+        if (this.useDash && this.defaultQuality !== 'auto') {
+          playerBandwidthOption.bandwidth = this.selectedBitrate * VHS_BANDWIDTH_VARIANCE + 1
```

Comment:
My habit to add code comment about references to doc
```suggestion
          // https://github.com/videojs/http-streaming#vhsbandwidth
          playerBandwidthOption.bandwidth = this.selectedBitrate * VHS_BANDWIDTH_VARIANCE + 1
```